virttest/utils_test/qemu/__init__.py

```diff
@@ -698,3 +699,308 @@ def get_all_memorys(self, vm):
         session = self.get_session(vm)
         output = session.cmd_output_safe(cmd, timeout=90)
         return set([_ for _ in output.splitlines() if _])
+
+
+class UsbDevTest(object):
+
+    """
+    Provide basic test for usb devices
+    """
+
+    def __init__(self, test, params, env):
+        self.params = params
+        self.test = test
+        self.env = env
+        self.vm = env.get_vm(params["main_vm"])
+
+    def check_usb_dev_monitor(self, usb_type):
+        """
+        check if the usb device exist in monitor
+        """
+        logging.info("Verify USB device in monitor.")
+        o = self.vm.monitor.info("usb")
```

Comment:
better to usb qmp commands, details see this post:
https://lists.nongnu.org/archive/html/qemu-devel/2014-07/msg02087.html